_IBC/aura-axelar.json

```diff
@@ -2,22 +2,22 @@
     "$schema": "../ibc_data.schema.json",
     "chain_1": {
       "chain_name": "aura",
-      "client_id": "07-tendermint-11",
-      "connection_id": "connection-8"
+      "client_id": "07-tendermint-17",
+      "connection_id": "connection-11"
     },
     "chain_2": {
       "chain_name": "axelar",
-      "client_id": "07-tendermint-117",
-      "connection_id": "connection-106"
+      "client_id": "07-tendermint-152",
+      "connection_id": "connection-137"
     },
     "channels": [
       {
         "chain_1": {
-          "channel_id": "channel-1",
+          "channel_id": "channel-4",
           "port_id": "transfer"
         },
         "chain_2": {
-          "channel_id": "channel-74",
```

Comment:
Do we have any proof that there are no stranded tokens that have been transferred on the original connection? or that both chains have voted to switch channels? or that the original connection should never have been registered?
The protect users' transferred funds, the registered canonical IBC connection cannot be changed unless voted by governances.

Reply:
The original connection had an incorrect unbonding time, it was not used as that makes the channel itself unusable already.

It was changed within 1-2 hours of the original one being created, just never updated here.

The original one shouldnt have been registered here eventhough it was considered the correct one for a very short time it didnt have funds travel over it.